src/lib/maybe-hydrate.js

```diff
@@ -29,11 +29,14 @@ module.exports = function maybeHydrate (inventory, callback) {
       if (!quiet) console.log(chalk.grey(chars.done, 'Found new functions to hydrate!'))
     }
     let notified = false
-    let shared = join(process.cwd(), 'src', 'shared') // TODO: impl inventory
-    let views = join(process.cwd(), 'src', 'views') // TODO: impl inventory
     // Make a new array, don't inventory
     let lambdaSrcDirs = [ ...inv.lambdaSrcDirs ]
-    lambdaSrcDirs.push(shared, views)
+
+    let shared = inv.shared && inv.shared.src || join(cwd, 'src', 'shared')
+    let views = inv.views && inv.views.src || join(cwd, 'src', 'views')
+    if (shared) lambdaSrcDirs.push(shared)
+    if (views) lambdaSrcDirs.push(views)
```

Comment:
Won't `shared` and `views` always be defined here as it'll fall back to them equaling `join(cwd, 'src', 'shared')` or `join(cwd, 'src', 'views')`? If so then the `if` conditional here is not needed, right?

Reply:
Aye, good catch, this was in a weird red herring bug path, I guess I glossed over tidying up that logic!